cypress/support/fragments/settings/users/usersOwners.js

```diff
@@ -29,8 +29,9 @@ const deleteOwner = (rowNumber = 2) => {
   cy.expect(Modal('Delete Fee/fine owner').absent());
 };
 
-export const getNewOwner = () => ({
-  owner: getTestEntityValue(),
+export const getNewOwner = (name, desc) => ({
+  owner: name ? getTestEntityValue(name) : getTestEntityValue(),
+  desc: desc ? getTestEntityValue(desc) : getTestEntityValue(),
```

Comment:
too complicated, getTestEntityValue already has such ternary operator

Reply:
changed to one call of the function getTestEntityValue(param)